tests/phpunit/tests/functions/getNonCachedIds.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Test `_get_non_cached_ids()`.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Test class for `_get_non_cached_ids()`.
+ *
+ * @group cache
+ *
+ * @covers ::_get_non_cached_ids
+ */
```

Comment:
```suggestion
/**
 * Test class for `_get_non_cached_ids()`.
 *
 * @package WordPress
 *
 * @group cache
 *
 * @covers ::_get_non_cached_ids
 */
```
Removing duplicate DocBlock for consistency with ther files.